tests/conftest.py

```diff
@@ -288,6 +316,19 @@ def pvc_factory_class(
     )
 
 
+@pytest.fixture(scope='session')
+def pvc_factory_session(
+    request,
+    storageclass_factory_session,
+    project_factory_session
+):
+    return pvc_factory_fixture(
+        request,
+        storageclass_factory_session,
+        project_factory_session
```

Comment:
@fbalak for new test cases, IMO, it would be better to use the default sc and Blockpool.

Reply:
I agree with @nehaberry here, I dont think we have to deal with fixtures here.

Reply:
In the PR I used pods in factory scope so I added ability to call pod_factory in session scope. After that I added ability to use session scope for all factory fixtures (like pvc_factory_session) as it didn't make sense to me to add only one session scoped factory. In test code I use default storageclasses.